packages/templates/clients/websocket/javascript/README.md

```diff
@@ -0,0 +1,9 @@
+## JavaScript WebSocket Client
+
+You can test this template:
+1. Clone the project and run `npm install`
+2. Navigate to `packages/templates/clients/websocket/javascript`
+3. Install with `npm install` and run test with `npm run test`
+4. Start example script that uses a client library generated by the test: `node example.js`
+
+> By default this is testing against Postman echo service. You can modify `packages/templates/clients/websocket/javascript/example.js` and change first line to `const WSClient = require('./tests/temp/snapshotTestResult/client-hoppscotch.js');` and run `node example.js` again. You will see example still works but agains different API. This is possible as both AsyncAPI documents have the same name of operation for sending messages: `sendEchoMessage` so each client generated has the same API.
```

Comment:
```suggestion
> By default this is testing against Postman echo service. You can modify `packages/templates/clients/websocket/javascript/example.js` and change first line to `const WSClient = require('./tests/temp/snapshotTestResult/client-hoppscotch.js');` and run `node example.js` again. You will see example still works but now it is using a different API. This is possible since both AsyncAPI documents define the same name of operation for sending messages: `sendEchoMessage` so each client generated has the same API.
```